AUTHORS

```diff
@@ -4,3 +4,4 @@ Artem Chernenkiy <notthewhite@gmail.com>
 Alexey Orlenko <eaglexrlnk@gmail.com>
 Mykola Bilochub <nbelochub@gmail.com>
 Vlad Dziuba <dzyubavlad@gmail.com>
+lundibundi <shishugi@gmail.com>
```

Comment:
@lundibundi I need your real name latinized ;)
It would also be nice if you set it in your Git config too.